packages/ui-react/src/pages/ScreenRouting/mediaScreens/MediaMovie/MediaMovie.tsx

```diff
@@ -70,18 +71,24 @@ const MediaMovie: ScreenComponent<PlaylistItem> = ({ data, isLoading }) => {
     return nextItem && navigate(mediaURL({ media: nextItem, playlistId: features?.recommendationsPlaylist, play: true }));
   }, [id, playlist, navigate, features?.recommendationsPlaylist]);
 
-  // Effects
   useEffect(() => {
     (document.scrollingElement || document.body).scroll({ top: 0 });
+    (document.querySelector('#video-details button') as HTMLElement)?.focus();
   }, [id]);
 
   // UI
   const pageTitle = `${data.title} - ${siteName}`;
   const canonicalUrl = data ? `${window.location.origin}${mediaURL({ media: data })}` : window.location.href;
 
-  const primaryMetadata = formatVideoMetaString(data);
+  const primaryMetadata = <VideoMetaData attributes={createVideoMetadata(data)} />;
   const shareButton = <ShareButton title={data.title} description={data.description} url={canonicalUrl} />;
-  const startWatchingButton = <StartWatchingButton item={data} playUrl={mediaURL({ media: data, playlistId: feedId, play: true })} />;
+  const startWatchingButton = (
+    <StartWatchingButton
+      key={id} // necessary to fix autofocus on TalkBack
```

Comment:
Because the OTT Web App is an SPA, it can be difficult for screen reader users to realize that the page has been navigated after clicking a card. 

To mitigate this issue, the focus can be reset to the first part that is changed after the route. We chose to focus the first interactive, but alternatively, we could focus the `<main />` or `<h1>` element instead.

@langemike @royschut can you elaborate on why we chose to focus the Start watching button?

Reply:
It seemed like the most logical element above the fold, making it the quickest way for a user to instantly play a video.

The focus will only be set on this button when the video content on the page switches. This means it only happens when you choose a video from within the CardGrid. It doesn't happen when you navigate to the page.

The alternatives are also viable options.